web/experimental/new_playground/styles.scss

```diff
@@ -532,6 +532,11 @@ a {
   }
 }
 
+.mdc-list-item__icon {
```

Comment:
The Material components have a built-in CSS class for the leading image in a list item, `mdc-list-item__graphic`:

https://github.com/material-components/material-components-web/tree/master/packages/mdc-list#css-classes

It would be good to reuse this, but it has the unfortunate side effect of trying to make a square out of our rectangular Flutter logo.

Consider doing this:

1) Remove this declaration and replace it with:

```
.mdc-list-item__graphic {
    height: auto;
}
```

That'll remove the squareness problem.

2) Use `mdc-list-item__graphic` instead of  `mdc-list-item__icon` when making the element above.